### DISCUSSION TITLE
What does sufixs K, T, W mean?

### DISCUSSION THREAD

**Discussion by adinvadim** [gcanti/fp-ts#1556] [Q&A] [answered]
I didn't find doc or arcticle about this. I want to understand fp-ts deeper.

I intuitively got, that "W" - it is mean "less strict", when one of generics accumulate result.

What I can read and study on this topiс?

**Comment by samhh** [comment, marked as answer]
They're described [here](https://gcanti.github.io/fp-ts/guides/code-conventions.html#what-a-t-suffix-means-eg-sequencet).

**Reply by adinvadim** [reply]
Thank you!
